enduser/2021/README.md

```diff
@@ -0,0 +1,69 @@
+# CNCF End User Community Survey 2021 Results
+
+This survey was conducted with the [CNCF End User Community](https://www.cncf.io/people/end-user-community/) between August 23 - September 13, 2021.
+
+The goal was to look at the programs and ensure they meet the needs of the End User Community for 2022.
+
+The results are also [available on Google Slides](https://docs.google.com/presentation/d/1YO1n-N16ObskUQ54Zchn-fnAmcn715ZKMvs6_9plUPA/edit?usp=sharing)
+
+# Summary
+
+![summary](images/summary.jpg)
+
+## Takeaways and action items
+
+- The Pandemic has contributed to more participant in activities virtually
+- Member Organizations want to increase Open Source Contributions with Guidance from the CNCF
```

Comment:
Change "Member Organizations" to "Ens User Organizations". Not all End Users are members, some of them are supporters